pages/concepts/ai-engine/ai-engine-personalities.mdx

```diff
@@ -0,0 +1,58 @@
+
+AI Engine personalities are not primarily about user experience or preference. Instead, they represent different versions or iterations of the AI Engine itself, each with its own set of features and capabilities.
+
+The primary reason for introducing different personalities is to manage the development and rollout of new features without disrupting existing functionality. By creating a new personality, developers can experiment with and test new features without affecting the core system. Once these features are stable and mature, they can be integrated into the default personality.
+
+## A Journey through personality types evolution
+
+The following AI Engine personality types are available for users: **Talkative V3** and **NextGeneration**.
+
+    - With version **v0.22**, the NextGeneration personality received significant updates, including support for classical UAgentResponse type agents, secondary function execution, and interactive UI elements. Notably, Talkative V3 was set to be replaced by NextGeneration in future updates.
+    - Version **v0.23** introduced enhanced session-level memory, tracking credit usage, parallel execution in sync worker, and session termination control within the NextGeneration personality.
+    - Version **v0.24** deployed **NextGeneration personality as the default**, and thus replacing **Talkative V3** personality type. This version focuses on significant stability and scalability improvements, supporting millions of agents. This version also introduced immediate agent registration, enhanced message reliability, UI enhancements, privacy compliance, robust timeout and monitoring systems, infrastructure upgrades, and database and deployment improvements.
+
+## Exploring the Feature Set of AI Engine Personalities
+
+Here's a breakdown of the key functionalities offered by each personality:
+
+### Talkative V3 - AI Engine V1
+
+    **Focus**: General conversation and task handling.
+
+    **Key Functionalities**:
+        - Enhanced task recommendations for improved efficiency.
+        - Improved context building for natural conversation flow.
+        - Agent health checks to ensure optimal performance.
+
+### NextGeneration - AI Engine V2
+
+    **Focus**: Broad conversational capabilities, advanced functionalities, and scalability.
+
+    **Key Functionalities**:
+        - Interacts with websites and PDFs directly.
+        - Tracks resource usage for transparency.
+        - Increased processing speed for faster responses.
+        - Manages session endings efficiently.
+        - Offers superior stability and scalability for large-scale applications.
+        - Completely revamped AI Engine backend for improved performance and future-proofing.
+        - Utilizes a more sophisticated LLM for enhanced context understanding.
+        - Employs an advanced recommender system to select optimal functions based on your needs.
+        - Supports a variety of dialogue protocols for flexible user interactions.
```

Comment:
As Attila suggested above I would emphasise that the main feature of next gen personality is that it supports dialogs (in opposite to the talkative personality above). So probably might be a good idea to move this line in the first bullet point and emphasise it as the most important feature.

Reply:
And can we please reference the documentation page which is about the concept of agent dialogues? If there is any documentation :D

Reply:
Yes we have a guide for this

Reply:
Will do